### PR TITLE
Update dockerfiles to only pull image from docker hub

### PR DESCRIPTION
**Pull request by yophilav**
Update dockerfiles to only pull image from docker hub

### CODE REVIEW

File: test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ARG base_tag=1.0.6.0-linux-arm32v7
-FROM edgebuilds.azurecr.io/microsoft/azureiotedge-module-base-full:${base_tag}
+FROM azureiotedge/azureiotedge-module-base-full:${base_tag}
```
Comment (philipktlin) [inline test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile:2]: please make sure all base images exist in docker hub, especially module base full.

**Comment by philipktlin** [comment]
please also find related script/yaml that pass base_registry argument to docker build and remove that argument from docker command.

File: test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile (hunk 1 above)
Comment (yophilav) [reply test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile:2]: Yeah, I'm going through building the images right now. 
It should fail then if an image doesn't exist.

**Comment by yophilav** [comment]
> please also find related script/yaml that pass base_registry argument to docker build and remove that argument from docker command.

Sure thing. I'll also update the base image update doc to be coherent with our current script.

File: scripts/linux/createArmBase.sh [hunk 2]
```diff
@@ -142,7 +142,7 @@ process_args()
         print_help_and_exit
     fi
 
-    registries=("edgebuilds.azurecr.io/" "edgerelease.azurecr.io/" "")
+    registries=("")
```
Comment (philipktlin) [inline scripts/linux/createArmBase.sh:145]: screen this script file to update comments, e.g. line7-8.

Comment (yophilav) [reply scripts/linux/createArmBase.sh:145]: Updated.

**Comment by and-rewsmith** [comment]
Have you already pushed all the new base images?

**Pull request merged**